protocol/avdl/keybase1/phone_numbers.avdl

```diff
@@ -0,0 +1,19 @@
+@namespace("keybase.1")
+protocol phoneNumbers {
+    import idl "common.avdl";
+
+    record PhoneNumberAddedMsg {
+        @jsonkey("phone")
+        string phoneNumber;
+    }
+
+    record PhoneNumberVerifiedMsg {
+        @jsonkey("phone")
+        string phoneNumber;
+    }
+
+    record PhoneNumberSupersededMsg {
+        @jsonkey("phone")
+        string phoneNumber;
+    }
```

Comment:
BTW you can also just do one msg for all 3, like `record PhoneNumberMsg` or something like that. Your call.

Reply:
yeah i mentioned that in the PR description. i initially did that, and thought the front end might want to treat them very differently (e.g. add phone number, don't care. superseded phone number: notify right away). i'm extremely indifferent.